test/DacpacTool.Tests/ScriptParserTests.cs

```diff
@@ -162,5 +162,16 @@ public void CanGenerateScriptWithNestedIncludesContainingRelativePaths()
             result.ShouldNotBeNull();
             result.ShouldMatch(@"PRINT N'Pre deploy'[\r\n]*PRINT N'Script1.sql'[\r\n]*PRINT N'Script4.sql'[\r\n]*PRINT N'Script5.sql'[\r\n]*PRINT N'Script6.sql'[\r\n]*GO[\r\n]*");
         }
+
+        [TestMethod]
+        public void ParserFailsWhenSyntaxError()
+        {
+            // Arrange
+            var collector = new ScriptParser($"{TEST_FOLDER}/TestProjectWithIncludedSyntaxErrors/Post-Deployment/Script.PostDeployment.sql", _variableResolver);
+
+            // Act / Assert
+            Should.Throw<System.InvalidOperationException>(() => collector.CollectFileNames())
+                .Message.ShouldBe("Incorrect syntax was encountered while parsing '\n'. File: DataScript.sql");
```

Comment:
The message here feels a bit weird to me, especially with the `\n` before the dot. Is that deliberate?

Reply:
The first half of the error comes from the test scenario I imported, per my previous comment, I just matched exactly what was coming back from the parser error.